package.json

```diff
@@ -30,6 +30,7 @@
     "pagespeed": "node ./internals/scripts/pagespeed.js",
     "presetup": "npm i chalk shelljs",
```

Comment:
Not necessarily, yarn can be installed with `npm i -g yarn` as per instructions in the readme file (we have a link there to latest install instructions for different OS).